gp2gp-translator/src/test/java/uk.nhs.adaptors.pss.translator/Generator/BundleGeneratorTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.nhs.adaptors.pss.translator.Generator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import uk.nhs.adaptors.pss.translator.generator.BundleGenerator;
+import uk.nhs.adaptors.pss.translator.service.FhirIDGeneratorService;
+import uk.nhs.adaptors.pss.translator.utils.ResourceTestFileUtils;
+
+@SpringBootTest
+@ExtendWith({SpringExtension.class})
+@DirtiesContext
+public class BundleGeneratorTest {
+
+    private static final FhirIDGeneratorService fhirIDGeneratorService = new FhirIDGeneratorService();
```

Comment:
This should be a mock (then `@ExtendWith(MockitoExtension.class)` class annotation is also needed)